src/routes/__layout.svelte

```diff
@@ -5,13 +5,16 @@
   import Header from '../components/Header.svelte'
   import type { Chapter, Link, NavLink } from '../types'
   import { fetchChapters, fetchYaml } from '../fetch'
+  import { microcopy } from '../stores'
   import '../app.css'
 
   export const load: Load = async () => {
     const nav = await fetchYaml(`Nav`)
     const footer = await fetchYaml(`Footer`)
     const social = await fetchYaml(`Social`)
     const chapters = await fetchChapters()
+    const smallTexts = await fetchYaml(`smallTexts`)
+    microcopy.set(smallTexts)
```

Comment:
We can use Svelte shorthand notation here:

```svelte
$microcopy = smallTexts
```

Reply:
In this script area, the shorthand notation is not possible. I'm getting the error message:
Cannot reference store value inside <script context="module">

Reply:
Ah wasn't aware of this limitation. Good to know.